### PR TITLE
feat: minio

### PR DESCRIPTION
**Pull request by srodenhuis**
Adds Minio to the Otomi apps with:
- option to provision buckets and policies for other Otomi apps
- OIDC prep
- configure persistence in Otomi values

### CODE REVIEW

File: helmfile.d/helmfile-07.init.yaml [hunk 1]
```diff
@@ -18,3 +18,9 @@ releases:
     labels:
       pkg: cert-manager
     <<: *raw
+  - name: minio-artifacts
+    installed: {{ $a | get "minio.enabled" }}
```
Comment (j-zimnowoda) [inline helmfile.d/helmfile-07.init.yaml:22]: since there are defaults you can simply do
```
$a.minio.enabled
```

File: values-schema.yaml [hunk 2]
```diff
@@ -2154,6 +2154,30 @@ properties:
...
+              enabled:
+                type: boolean
+                default: true
+          persistence:
+            properties:
+              enabled:
+                type: boolean
+                default: true
+              size:
+                type: string
```
Comment (j-zimnowoda) [inline values-schema.yaml:2179]: It is better to use the type that has already been defined for size: `$ref: '#/definitions/size'`

File: helmfile.d/helmfile-07.init.yaml (hunk 1 above)
Comment (srodenhuis) [reply helmfile.d/helmfile-07.init.yaml]: fixed

**Review by j-zimnowoda — APPROVED** [review]
:)

File: values-schema.yaml (hunk 2 above)
Comment (srodenhuis) [reply values-schema.yaml]: fixed

**Pull request merged**